utest/py3270/test_emulator.py

```diff
@@ -50,54 +50,6 @@ def test_emulator_none_windows_visible(mock_posix):
     assert under_test.app.executable == "x3270"
 
 
-def test_emulator_with_extra_args_oneline(mock_windows):
```

Comment:
These need to be deleted because the processing of the args is now handled by x3270.py